sagemaker-pyspark-sdk/tests/wrapper_test.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pytest
+
+from pyspark import SparkConf, SparkContext
+
+from sagemaker_pyspark import (S3DataPath, EndpointCreationPolicy, RandomNamePolicyFactory,
+                               SageMakerClients, IAMRole, classpath_jars)
+from sagemaker_pyspark.wrapper import Option, ScalaMap, ScalaList
+
+
+@pytest.fixture(autouse=True)
+def with_spark_context():
+    os.environ['SPARK_CLASSPATH'] = ":".join(classpath_jars())
+    conf = (SparkConf()
+            .set("spark.driver.extraClassPath", os.environ['SPARK_CLASSPATH']))
+
+    if SparkContext._active_spark_context is None:
+        SparkContext(conf=conf)
+
+    yield SparkContext._active_spark_context
+
+    # TearDown
+    SparkContext.stop(SparkContext._active_spark_context)
+
+
+def test_convert_dictionary():
```

Comment:
Nice! good job adding more coverage to such an important part of our wrapper :)

Reply:
Thank you!